lib/pana.dart

```diff
@@ -46,38 +44,32 @@ class PackageAnalyzer {
   Future<Summary> inspectPackage(
     String package, {
     String version,
-    String packageDir,
     bool keepTransitiveLibs: false,
   }) async {
-    var toolProblems = <ToolProblem>[];
-    var sdkVersion = _dartSdk.version;
-    log.info("SDK: $sdkVersion");
-
-    log.info("Package: $package");
+    _logEnv();
```

Comment:
Also ends up calling _inspect – perhaps _logEnv should just be within _inspect...